tests/queries/0_stateless/02597_projection_materialize_and_replication.sql

```diff
@@ -14,13 +14,20 @@ set mutations_sync=0;
 
 ALTER TABLE test UPDATE d = d || toString(sleepEachRow(0.3)) where 1;
 
-ALTER TABLE test ADD PROJECTION d_order ( SELECT min(c_id) GROUP BY `d`);
-ALTER TABLE test MATERIALIZE PROJECTION d_order;
-ALTER TABLE test DROP PROJECTION d_order SETTINGS mutations_sync = 2; --{serverError BAD_ARGUMENTS}
```

Comment:
Based on the [PR](https://github.com/ClickHouse/ClickHouse/pull/50181/files#diff-2001369ec75a93211af30ee6dc47c9903e9a3171f5085ca24db8ebbe386860bb) that introduced this test this exception is the gist of the test. Removing it basically makes the tests useless.

Reply:
After talking in private we discussed that these changes might lead to a test that is not checking anymore what is was meant to check. Therefore we had two options:
1. remove the test
2. try to make it less flaky

We opted for no2 by increasing the length of the mutation. Hopefully this will reduce the flakyness of the tests. Theoretically it can still fail, but hopefully the increase runtime of the mutation will reduce its flakyness very close to zero (it failed 0 times out of 100 in flaky test check). The length of the test increased to 30 seconds in worst case, but unfortunately we couldn't come up with a better solution.